_pages/fr/faq.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title:  "Frequently Asked Questions"
+permalink: /lang/faq
```

Comment:
since this is the FR version, please already replace "lang" with "fr" (same for the lang: property below)